ui/components/ConfirmationModal.js

```diff
@@ -89,9 +89,9 @@ const styles = (theme) => ({
     margin : theme.spacing(0.5),
     padding : theme.spacing(1),
     borderRadius : 5,
-    backgroundColor : "#ff3333",
+    backgroundColor : "#B32700",
```

Comment:
@legendarykamal I think you should avoid adding the color like this , refer to the predefined file to use the color.